src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingTextDocumentEventArgsWrapper.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal readonly struct UnitTestingTextDocumentEventArgsWrapper
+    {
+        internal TextDocumentEventArgs UnderlyingObject { get; }
```

Comment:
Should this be private? I can't access this on Unit Testing side, can I?

Reply:
I kept this consistent with other such wrapper types in the EA layers. All of them have an internal UnderlyingObject property.